docs/pages/docs/01-cloud-deployment/aws-deployment.md

```diff
@@ -0,0 +1,131 @@
+---
+title: AWS Deployment
+keywords: cloud, deployment, aws
+tags: [getting-started, deployment, cloud]
+summary: "Deploying Grakn on AWS"
+sidebar: documentation_sidebar
+permalink: /docs/cloud-deployment/aws-deployment
+folder: docs
+---
+
+# AWS Marketplace
+
+## Deployment
+
+TBC
+
+### Stack parameters
+The following parameters allow to define the stack details:
+
+* General:
+    - **StackName**
+    
+        Name for you Grakn KGMS stack. This must be a valid system name, specifically, it must consist of only lowercase letters, numbers, and hyphens and cannot exceed 50 characters. 
+      
+    - **KeyPairName**
+        
+        The key pair assigned to Grakn instances to allow SSH access.
+    
+* Network:
+    - **VPC**
+    
+        The id of the VPC the stack should be deployed into.
+    - **InternetFacingRouteTable**
+      
+        The id of a internet-facing Route Table in the VPC.
+    - **GraknSubnetCidrBlock**
+   
+        The Grakn cluster will be deployed in a separate subnet and this setting specifies its IP CIDR range.
+    - **SSHLocation**
+     
+        The IP address range that can be used to access Grakn instances using SSH.
+    - **gRPCLocation**
+   
+        The IP address range that can be used to make gRPC requests.
+     
+
+* Cluster:
+    - **GraknInstanceType**
+    
+        EC2 instance type of a single Grakn instance.
+    - **GraknGroupSize**
+    
+        Number of Grakn instances in the cluster.
+    - **EbsVolumeSize**
+    
+        Size in GB of each of EBS volumes that are attached to Grakn instances.
+    - **OptimiseForEbs**
+    
+        Specifies whether the Grakn launch configuration should optimised for EBS I/O.
+        
+## Accessing Grakn
+
+There are various ways to access Grakn on AWS. Here we will address the most common usage patterns.
+
+### Using gRPC
+
+The most common access pattern is to use a gRPC client to connect to a Grakn cluster. The connection happens via TCP port 48555 and the `gRPCLocation` stack parameter defines
+the range of IP addresses that can schedule gRPC requests.
+        
+### Logging into a node
+To log in into one of the cluster nodes, simply use the ssh command:
+
+`ssh -i <private key file> ubuntu@<grakn instance DNS name or IP address>`
+
+#### Cluster health check
+To verify cluster state, execute the following command:
+    
+`grakn cluster status`
+     
+A sample output of the command executed on a node of a healthy 3-node cluster shall look similar to this:
+
+![](/images/aws-cluster-health.png)
+
+#### Accessing the Graql console
+
+To access the Graql console, a user password is required. The default user password is specified in the `GraknUserPassword` output.
+
+To log into the Graql console, simply type `graql console`. After entering the user credentials (user: grakn, password: the one from Google console), you are free to interact with Grakn via the Graql terminal. 
```

Comment:
perhaps:

...(user: grakn, password: the one specified in the `GraknUserPassword` output),...

is more accurate?